drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-showcase/src/main/resources/example-repositories.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2016 Red Hat, Inc. and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+drools-playground=https://github.com/guvnorngtestuser1/guvnorng-playground.git
+jbpm-playground=https://github.com/guvnorngtestuser1/jbpm-console-ng-playground-kjar.git
```

Comment:
These should not be needed anywhere. I'll add the original to my clean up list. Can you remove this one?

Reply:
Fixed in
[DROOLS-2786] Investigate and create a scenario simulation showcase - Removing unneeded repository properties